readme.md

```diff
@@ -142,6 +142,14 @@ Feel free to write to our [Support Center](http://devexpress.com/support/center)
     - [Change the Color of Legend Item Icons](#change-the-color-of-legend-item-icons)
     - [Change the Tooltip Z-Index](#change-the-tooltip-z-index)
     - [Set the Tooltip Background Color](#set-the-tooltip-background-color)
+  - [DxUpload](#dxupload)
+    - [Change Select Button Color](#change-select-button-color)
+    - [Change Color of Labels with File Sizes](#change-color-of-labels-with-file-sizes)
```

Comment:
What do you mean by "_with_ File Sizes"? For example, do you mean "**based on** File Size"?

Reply:
I mean the following text string:
![image](https://github.com/DevExpress-Examples/blazor-style-migration/assets/80316217/42ca05f4-781a-4175-bc70-852872623caf)

Reply:
I see now. I think we can update it as follows:

```suggestion
    - [Change Color of File Size Labels](#change-color-of-labels-with-file-sizes)
```

Reply:
```suggestion
    - [Change Color of File Size Labels](#change-color-of-file-size-labels)
```